index.html

```diff
@@ -355,15 +354,13 @@
         <ol>
           <li>If the screen width is greater than the screen height, set the
           <a>document</a>'s <a>current orientation type</a> to
-            <code><dfn for='OrientationType'>landscape-primary</dfn></code>
-            or <code><dfn for=
-            'OrientationType'>landscape-secondary</dfn></code>.
+          <dfn data-dfn-for='OrientationType'>landscape-primary</dfn> or
```

Comment:
Let's move the `data-dfn-for='OrientationType'` to the `section` element, then it will just work with `<dfn>`